src/index.less

```diff
@@ -1,476 +1,338 @@
+          color: @sidebar-actions-menu-item-font-color !important;
         }
 
-        .item-prefix {
-          margin-right: 4px;
-          font-size: 10px;
+        .item-title {
+          color: @sidebar-actions-menu-item-font-color !important;
         }
 
-        &:hover,
         &:active,
+        &:hover,
         &:focus {
-          text-decoration: none;
+          .item-icon {
+            background-color: @sidebar-actions-menu-item-background-color_hover !important;
+            color: @sidebar-actions-menu-item-font-color_hover !important;
+          }
         }
       }
+    }
+  }
 
-      /**
-      * 'white-space: nowrap' is causing text overflow to not be hidden in
-      * firefox, this fix the issue at the cost of loosing text ellipsis...
-      * any better solution is welcome
-      */
-      @-moz-document url-prefix() {
-        .menu-item {
-          white-space: normal !important;
-          word-break: break-all !important;
-        }
+  .menu-item {
+    width: 100%;
+    padding: @sidebar-item-padding;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    background-color: transparent;
+    border: 0;
+    color: @sidebar-item-font-color;
+    font-weight: @sidebar-item-font-weight;
+    text-align: left;
+    text-decoration: none;
+
+    .item-arrow,
+    .item-loading,
+    .item-icon,
+    .item-prefix,
+    .item-title {
+      line-height: @sidebar-item-line-height;
+      display: inline-block;
+    }
+
+    .item-loading,
+    .item-arrow {
+      width: @sidebar-item-arrow-width;
+      margin-right: @sidebar-item-arrow-spacing;
+      font-size: @sidebar-item-arrow-size;
+      text-align: center;
+    }
+
+    .item-icon {
+      width: @sidebar-item-icon-width;
+      font-size: @sidebar-item-icon-size;
+      margin-right: @sidebar-item-icon-spacing;
+      text-align: center;
+    }
+
+    .item-prefix {
+      margin-right: @sidebar-item-prefix-spacing;
+      font-size: @sidebar-item-prefix-size;
+    }
+
+    .item-title {
+      flex: 1;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+
+    .oui-spinner {
+      vertical-align: middle;
+    }
+
+    &:focus {
+      outline-width: 1px;
+      outline-style: dotted;
+      outline-color: initial;
+      outline-offset: -2px;
+    }
+
+    &:hover,
+    &:active,
+    &:focus {
+      text-decoration: none;
+
+      .item-title {
+        text-decoration: underline;
       }
+    }
+  }
 
-      .menu-view-all {
-        .menu-view-all-inner {
-          border-top: 1px solid @SIDEBAR_MENU_VIEW_ALL_ITEM_BORDER;
+  .active > .menu-item {
+    color: @sidebar-item-font-color_active;
+    background: @sidebar-item-background-color_active;
+    font-weight: @sidebar-item-font-weight_active;
+  }
 
-          a {
-            font-weight: bold;
-            letter-spacing: 0.02em;
-          }
+  .menu-view-all {
+    .menu-view-all-inner {
+      border-top: 1px solid #8392a4;
```

Comment:
This color doesn't deserve a variable?

Reply:
I can't reproduce the use of this part in the Managers. So I didn't touch its properties and values ;)